### DISCUSSION TITLE
how to enter into uniffle wechat or dingtalk?

### DISCUSSION THREAD

**Discussion by qin7972** [apache/celeborn#1475] [Q&A]
Hi,
I want to enter into uniffle wechat or dingtalk，how to enter?

**Comment by FMX** [comment]
If you are looking for a shuffle solution, trying Celeborn will be a great idea.

You can ask questions in the Uniffle community to get the contact info.

**Comment by FMX** [comment]
Celeborn DingTalk Group : （  https://qr.dingtalk.com/action/joingroup?code=v1,k1,2OYn0jXy0SceP8CbLqiZUkFIOVtNOhtWbsi1oAUy9hw=&_dt_no_comment=1&origin=11  ）  or  41594456
